editor/src/core/property-controls/property-controls-local.ts

```diff
@@ -615,9 +638,10 @@ function componentInsertOptionFromExample(
           code: `<${typed.name} />`,
         }
       }
+      const jsxName = jsxElementNameFromString(typed.name)
       return {
         label: typed.name,
-        imports: `import {${typed.name}} from '${moduleName}'`,
+        imports: `import {${jsxName.baseVariable}} from '${moduleName}'`,
```

Comment:
is this going to work for antd Typography.Text -like stuff? Wouldn't this become `import { Text } from 'antd'` ?